### DISCUSSION TITLE
global search implementation in decouple cms orchard core

### DISCUSSION THREAD

**Discussion by saleemhaq** [OrchardCMS/OrchardCore#10955] [Q&A]
How to implement the global search implementation in decouple cms like below we have design:

![image](https://user-images.githubusercontent.com/5202291/147969086-b6d98898-0833-458b-97c2-a03ec3cd3f72.png)

**Comment by ns8482e** [comment]
Look at the documentation https://docs.orchardcore.net/en/latest/docs/topics/search/
